### DISCUSSION TITLE
Amazing Work, Many Thanks

### DISCUSSION THREAD

**Discussion by HarryHaller245** [tttapa/Control-Surface-Motor-Fader#20] [General]
Hey, many thanks for this.
I started my project as a complete beginner, and thanks to your great documentation and flexible code I was still able to use your Arduino sketches, and python scripts and also gain a deeper understanding of whats happening here. I have also only destroyed one raspberry pi in the process. As soon as I am finished with the Project or it is a little more presentable I will update and add Information that would have saved me some time :D

Well anyway, many thanks again!

**Comment by tttapa** [comment]
Glad to hear that it was useful! :)

**Reply by HarryHaller245** [reply]
Hello again,
I am very new to Github, and I didnt know where to post this.
But I have designed a PCB that uses your arduino skecth on an arduino nano.
Maybe someone finds this useful.

I only used the serial-midi functionality of the sketch,
therefore there are no explicit connectors for the arduino, except the USB port.
You could however solder to the unused pins.
I may update the design at some point.

I have attached the needed files to get it produced or modify it:
- EAGLE PCB file (.brd)
- ODB Files
- Parts List/BOM
- Picture of assebled PCB
- Snapshot of connector assignments

Quick Disclaimer:  
I have no Idea what I am doing.  
Build at your own risk obviously (mostly your money)

PCB Components/BOM:

	QTYx Description
		value
		PCB Name
		Device
		Footprint
	
	2x 1M Resistor
		1M
		R1,R2
		AXIAL-11.7MM-PITCH (R-US) 
		(Version 3)RESAD1176W63L850D250B
	
	2x 100nF Capacitor
		100nF
		C1,C2
		CHIP-0402(1005-METRIC)
		CAPC1005X60
		I used: # Kemet C0402C104K4RAC7867+ Keramik-Kondensator SMD 0402 100 nF 16 V
		It does work without this!
		
	2x DUPONT Connectors 90DG 1x2
		.
		.
		FMOT, FMOT1
		/90 (PINHD-1X2) (Version 4)
		1X02/90 (Version 1)
		
	2x DUPONT Connectors 90DG 1x4
		.
		.
		FPOT-TP, FMOT-TP1
		/90 (PINHD-1X4) (Version 4)
		1X04/90 (Version 1)
		
	1x Motorcontroller L293
		.
		L293
		
	1x Motorcontroller Socket L293 (Optional)
		
	1x Arduino Nano
	
	1x Arduino Nano Socket (Optional)

I used this Fader
	ALPS RSAON11M9 Motorised fader 10 kΩ 0.5 W linear 1 pc(s)
	Data Sheet: https://asset.conrad.com/media10/add/160267/c1/-/en/000442081DS01/datasheet-442081-alps-rsaon11m9-motorised-fader-10-k-05-w-linear-1-pcs.pdf

For a more elegant connector layout check .brd file:
Here is rough write down. (Top to bottom)
(The correct motor pin  is a 50/50 chance :) )

Connector assignments

	|      |
	12V   GND
	
	#### FMOT1
	MOT-P1 - (DC Motor 1)
	MOT-N1 -(DC Motor 2)
	
	#### FMOT-TP1
	D9-TPOINT - (Touchpoint of the Fader)
	5V - (5V of the potentiometer)
	A1-WIPER - ( wiper of the potentiometer)
	GND - (gnd of the potentiometer)
	
	#### FMOT
	MOT-P -
	MOT-N -
	
	#### FPOT-TP
	D9-TPOINT -
	5V -
	A1-WIPER -
	GND -
	
	#### USB
	||||||||



[motorizedFaderControllerv29.zip](https://github.com/tttapa/Control-Surface-Motor-Fader/files/14117188/motorizedFaderControllerv29.zip)

**Reply by tttapa** [reply]
Great, thanks for sharing!
